lfucg-exactions/client/js/components/AccountLedgerSummary.js

```diff
@@ -92,260 +96,33 @@ class AccountLedgerSummary extends React.Component {
-                                                                <i className="fa fa-pencil-square link-icon col-xs-4" aria-hidden="true" />
-                                                                <div className="col-xs-7 link-label">
-                                                                    Edit
-                                                                </div>
-                                                            </Link>
-                                                        }
-                                                    </div>
-                                                    <div className="col-xs-5 ">
-                                                        <Link to={`agreement/summary/${accountLedgers.agreement.id}`} aria-label="Summary">
-                                                            <i className="fa fa-file-text link-icon col-xs-4" aria-hidden="true" />
-                                                            <div className="col-xs-7 link-label">
-                                                                Summary
-                                                            </div>
-                                                        </Link>
-                                                    </div>
-                                                </div>
-                                            </div>
-                                            <div className="col-xs-12">
-                                                <p className="col-xs-6">Current Balance: {accountLedgers.agreement.agreement_balance && accountLedgers.agreement.agreement_balance.total}</p>
-                                                <p className="col-xs-6">Resolution Number: {accountLedgers.agreement.resolution_number}</p>
-                                                <p className="col-xs-6">Expansion Area: {accountLedgers.agreement.expansion_area}</p>
-                                                <p className="col-xs-6">Agreement Type: {accountLedgers.agreement.agreement_type_display}</p>
-                                                <p className="col-xs-6">Date Executed: {accountLedgers.agreement.date_executed}</p>
-                                            </div>
-                                        </div>
-                                    </div>
-                                </div>
-                            :
-                                <div className="row section-heading" role="tab" id="headingAccountPlats">
-                                    <h3>Agreement - None</h3>
-                                </div>
-                            }
+                            <LotsMiniSummary
+                              mapSet={accountLedgers.lot}
+                              mapQualifier={accountLedgers && accountLedgers.lot && accountLedgers.lot.id}
+                              singleLot={true}
```

Comment:
It looks like we get a string each time and to me it seemed like we weren't really getting out of checking for this... but after talking this logic is replacing the logic we previously had to wait on the page load for mapping.